_posts/2018-02-02-45-minutes-into-powerapps.md

```diff
@@ -0,0 +1,43 @@
+
+![](/img/powerapps/powerapps-dashboard.png)
+
+This is not to say RN is inherently bad, or even that MS's apparoach is wrong. It's just something to keep in mind if you're planning on making PowerApps an integral part of your business processes.
+
+## Going public - In organisational context
+
+After tweaks and rebranding efforts it’s time to publish. Hit File -> Save and you’ll be one click away from gentle scrutiny of your first app by your colleagues.  First time around it’ll ask for who can do what with your app, options are Edit or View. Going back to start page of PowerApps and hitting Apps will give you list of the Apps available for you. At this point the App has magically appeared into your mobile devices PowerApps app too. from list selecting “…” -> Settings presents you with App’s global settings. From where you can edit various things about the app, but interesting bitsare the Share and Versions tabs. From Share you’ll be able to control who can do what with your app. From versions you control, which version of the app is currently published for the audience. 
+
+There’s also an Export package button which allows you download the app package. App package is pretty much a save file meant be uploaded back to PowerApps, or maybe edit offline. Important part is that this app package has nothing to do with an actual mobile app in traditional context.
+
+Was that it? Pretty much yes. PowerApps presents you with ability to connect enterprise data sources, create UI to present the data and manage access to apps within your organisation. And it does all that in simple intuitive way. Obviously I merely scratched the surface here and deep down there you may actually find yourself writing an actual line of code, in JS, what else? After all, in extremely oversimplified definition, PowerApps is a ReactNative container for components and connectors. Should you use it for enterprise apps then? Absolutely, if you are willign to live in MS (and some 3rd party services) ecosystem, and you have rather simple tasks to solve. PowerApps offer great opportunity to take down the load of actually running an organisation. I’d be hesitant to actually try to sell these solutions to customers at least until they fully undesrstand implications and limitations presented by this type of approach. As a side note for what it's worth, it took me two times longer to write this post than to have the Contacts up and running shared to selection of colleagues to test.  
```

Comment:
with the ability to connect
create an UI
in a simple intuitive way
PowerApps offerS A great opportunity